### ISSUE TITLE
`no-invalid-interactive` does not trigger when using the `on` modifier

### ISSUE THREAD

**Issue by Windvis** [ember-template-lint/ember-template-lint#731]
The `no-invalid-interactive` rule doesn't throw an error when using the `on` modifier instead of the action helper / modifier.

```
    <div {{action this.actionName}}>...</div>{{!-- This triggers a lint error --}}
    <div {{on "click" this.actionName}}>...</div>{{!-- This does not --}}
```

**Comment by rwjblue** [comment]
Yep, we need to fix this! Thanks for reporting!

**Comment by lifeart** [comment]
on it

**Comment by lifeart** [comment]
PR created
